src/sass/_common/base/constants.scss

```diff
@@ -38,18 +37,8 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
-
-$COLOR_NETWORK_STATUS_ONLINE: #4bb4b3;
-$COLOR_NETWORK_STATUS_OFFLINE: #ec3f3f;
-$COLOR_NETWORK_STATUS_BLINKER: #4bb4b3;
-$COLOR_BORDER_GRAY: #f2f3f4;
-$COLOR_BALLOON_GRAY: #d6dadb;
-$COLOR_HOVER_FULLSCREEN_GRAY: #e6e9e9;
```

Comment:
why delete these? I'm using some of them, and the codebase needs the others too

Reply:
- I've removed `$COLOR_NETWORK_STATUS_BLINKER` and `$COLOR_HOVER_FULLSCREEN_GRAY` as they're not from Binary Static originally, I've updated their usages by using CSS vars from `theme.scss`. 
- `$COLOR_BORDER_GRAY` and `$COLOR_BALLOON_GRAY` should still be available, however we should try to use CSS vars where possible.

Reply:
I think we shouldn't delete them, even if we don't use them let's keep them. we want to change the codebase as less as possible

Reply:
They were added as part of our changes (Brandon's task).

Reply:
ah, I see! I had no idea :) it's alright then, thank you